src/AccessLog.php

```diff
@@ -105,6 +110,20 @@ public function hostnameLookups($hostnameLookups = true)
     }
 
     /**
+     * Set context callable
+     *
+     * @param bool $context
+     *
+     * @return self
+     */
+    public function context(callable $context = null)
```

Comment:
The type in phpdocs is wrong. And maybe makes no sense allowing null values here.

Reply:
Makes sense 👍